_employment/job-seekers/employment-support.md

```diff
@@ -32,55 +32,55 @@ If you are out of work, you may be eligible for [unemployment support](http://ca
 
         <li>
           <h3>
-            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance:</a>
+            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance</a>
           </h3>
         </li>
 
         <li>
           <h3>
-            <a href="https://www.vets.gov/veterans-employment-center/skills-translator">Skills Translator</a>
+            <a href="/employment/job_seekers/skills-translator">Skills Translator</a>
```

Comment:
@emilyville want to verify that this should be `job_seekers` and not `job-seekers`